Orange/widgets/model/owrules.py

```diff
@@ -349,6 +357,7 @@ def get_learner_parameters(self):
             ("Covering algorithm", self.storage_covers[self.covering_algorithm]),
             ("Gamma", self.gamma),
             ("Evaluation measure", self.storage_measures[self.evaluation_measure]),
+            ("Restrict equality", self.restrict_equality),
```

Comment:
I think here we should have "Restrict to equality" (which is analogous to what is in the Slovenian translation).